pom.xml

```diff
@@ -240,6 +240,18 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.apache.kyuubi</groupId>
```

Comment:
emm?

Reply:
Without this, the log4j-1.2.17 dependency will be transferred.

Reply:
to be clear, Do you mean that the common module needs log4j-1.2.17 but the server module not? so you need to exclustion?

Reply:
The kyuubi-common module is depended by other modules, such as kyuubi_assembly.

For command, `mvn dependency:tree -pl :kyuubi-common_2.12|grep 1.2.17`, the result is empty.


But if you execute below command:
```
mvn dependency:tree |grep  -C 20 1.2.17
```

You can find that the log4j-1.2.17 is from kyuubi-common module.

Reply:
So, I think there is some dependency transfer issue.


And declare the kyuubi-common in dependencyManagement can help fix this issue.

Reply:
why not exclude it in common？

Reply:
let me have a try.